### PR TITLE
Docker Compatibility List

### PR DESCRIPTION
**Pull request by sflxn**
This is a swag at initial docker support.  Right now, there's a scrub of the CLI commands and options.  There is also a scrub of Docker Compose file options.  This will be a set of living documents that will change over time as we figure out what the final list of parameters should be.

### CODE REVIEW

File: apiservers/engine/CLI-COMPATIBILITY.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
+# Docker CLI Compatibility
+VIC attempts to be compatible with Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that applies to running in an ESX or vCenter environment.
```
Comment (andrewtchin) [inline apiservers/engine/CLI-COMPATIBILITY.md]: s/applies/apply

File: apiservers/engine/CLI-COMPATIBILITY.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+The list is organized into separate 3 tables.  The first list CLI operations that is supported in the first version of VIC.  The second list are operations that VIC may support in the future.
```
Comment (andrewtchin) [inline apiservers/engine/CLI-COMPATIBILITY.md]: s/separate 3/3 separate

**Comment by andrewtchin** [comment]
lgtm

File: apiservers/engine/CLI-COMPATIBILITY.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
+# Docker CLI Compatibility
+VIC attempts to be compatible with Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that apply to running in an ESX or vCenter environment.
```
Comment (mdubya66) [inline apiservers/engine/CLI-COMPATIBILITY.md]: VIC attempts to be compatible with the Docker client (CLI); however, there are some differences.  The CLI differences are documented below, alphabetically.  Note, the operations listed as supported but with differences does not mean incompatible.  It means most of these operations support either a smaller subset of the parameters or the request/response parameters have different meanings that apply to running in an ESX or vCenter environment.

File: apiservers/engine/CLI-COMPATIBILITY.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+
+The list is organized into 3 separate tables.  The first list CLI operations that is supported in the first version of VIC.  The second list are operations that VIC may support in the future.
```
Comment (mdubya66) [inline apiservers/engine/CLI-COMPATIBILITY.md]: The list is organized into 3 separate tables.  The first table lists CLI operations that are supported in the first version of VIC.  The second table lists operations that VIC may support in the future.

File: apiservers/engine/CLI-COMPATIBILITY.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+```
+
+| params | Docker | VIC |
+|---|---|---|
+|-a, --attach=[]|Attach to STDIN, STDOUT or STDERR|YES|
+|--add-host=[] |Add a custom host-to-IP mapping (host:ip)|YES|
+|-c, --cpu-shares=0 | CPU shares (relative weight)|*diff*|
+|--cap-add=[]|Add Linux capabilities|NO, VIC containers do not run in cgroups|
+|--cap-drop=[]|Drop Linux capabilities|NO, VIC containers do not run in cgroups|
+|--cgroup-parent=""|Optional parent cgroup for the container|NO, VIC containers do not run in cgroups|
```
Comment (mdubya66) [inline apiservers/engine/CLI-COMPATIBILITY.md]: --cgroup-parent=""|Optional parent cgroup for the container|VIC containers do not run in cgroups|

File: apiservers/engine/CLI-COMPATIBILITY.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+### Response:
+
+The response for inspect is a JSON document composed of several inner documents.  The following are the primary subdocuments that differs.  For the rest of the response, please visit,
+```
+https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#inspect-a-container.
+```
+
+**AppArmorProfile**
+
+VIC containers isolation unit is a VM.  Processes running in a VIC container do not use AppArmor profiles.
```
Comment (mdubya66) [inline apiservers/engine/CLI-COMPATIBILITY.md]: A VIC container isolation unit is a VM.  Processes running in a VIC container do not use AppArmor profiles.

**Pull request merged**